open-metadata-implementation/governance-servers/integration-daemon-services/integration-daemon-services-server/src/main/java/org/odpi/openmetadata/governanceservers/integrationdaemonservices/handlers/IntegrationServiceHandler.java

```diff
@@ -254,11 +254,15 @@ public void shutdown()
     {
         final String actionDescription = "Server shutdown";
 
-        for (IntegrationConnectorHandler connectorHandler : connectorHandlers)
-        {
-            if (connectorHandler != null)
-            {
-                connectorHandler.shutdown(actionDescription);
+        /*
+        if a integration has encountered a problem when starting
+        then the connectorHandlers can be null
+         */
```

Comment:
Need to change the code formatting to match the rest of the file/module

Reply:
as requested .